packages/wdio-appium-service/src/launcher.ts

```diff
@@ -130,13 +128,20 @@ export default class AppiumLauncher implements Services.ServiceInstance {
          * Append remaining arguments
          */
         this._appiumCliArgs.push(...formatCliArgs(this._args))
-        this._setCapabilities()
+
+        /**
+         * Get port from service option or use a random port
+         */
+        const port = typeof this._args.port === 'number'
+            ? this._args.port
+            : await getPort({ port: DEFAULT_APPIUM_PORT })
+        this._setCapabilities(port)
```

Comment:
@christian-bromann it feels like you missed adding the new port value to the `_appiumCliArgs` arr...
```
this._appiumCliArgs.push(...formatCliArgs({ port }))
```

Without the line above, it tries to start the appium on port 4723, even logs say different port value...

Reply:
Mind raising a PR with a fix?